src/sentry/db/models/manager.py

```diff
@@ -325,4 +324,5 @@ def bind_nodes(self, object_list, *node_names):
 
         for item, node in object_node_list:
             data = node_results.get(node.id) or {}
-            node.bind_data(data, ref=node.get_ref(item))
+
+            node.bind_data(data, ref=node.get_ref(item), event_datetime=item.datetime)
```

Comment:
Does this work for transactions as well? They use different fields for the timestamp.

Reply:
Right now it will, since SnubaEvent always comes from the events table, might need to change when we extend SnubaEvent to support data fetched from transactions.